pkg/txn/impl/processor.go

```diff
@@ -420,6 +420,38 @@ func (b *batch) TxnReceiptExists(ctx context.Context, txnHash common.Hash) (bool
 	return exists, nil
 }
 
+// ChangeTableOwner changes the owner of the table in the registry table.
+func (b *batch) ChangeTableOwner(
```

Comment:
Opted for the `Owner` name instead of `controller` (name of the column). Looks like the name `controller` is now more related to the Controller smart contract.

Reply:
Q: Worth renaming the `registry` table column?

Reply:
I thought about it. I think it is but wouldn't do in this PR. There's some discussion [here](https://github.com/tablelandnetwork/go-tableland/issues/187) about how `controller` is misleading. Maybe we can take advantage of what will come up there to do this change.

Reply:
:+1: